indexer/scripts/elastic-conf.py

```diff
@@ -144,6 +154,35 @@ def create_initial_index(self, es: Elasticsearch) -> Any:
                 logger.error("Failed to create Index. Response:%s", response)
             return acknowledged
 
+    def create_slm_policy(self, es: Elasticsearch) -> Any:
+        _TODAY = date.today().strftime("%Y.%m.%d")
+        policy_id = f"slm-policy-{_TODAY}"
```

Comment:
```suggestion
        policy_id = "name of the file"
```

I think `policy_id` needs to be a constant. Otherwise, running it on different days will end creating multiple policies (and activate all of them?).